server/server.go

```diff
@@ -99,24 +102,24 @@ var casbinDeveloperPolicy string
 // @schemes http
 
 // NewServer creates a server.
-func NewServer(ctx context.Context, prof Profile) (*Server, error) {
+func NewServer(ctx context.Context, profile Profile) (*Server, error) {
```

Comment:
I don't have any opinion on either prof or profile, BUT less refactoring would be preferred to minimize the scope of the change. Refactoring should be done only when it brings in VALUE to R&D.